site/content/entry/citra-vulkan-announcement/index.md

```diff
@@ -0,0 +1,110 @@
++++
+date = "2023-08-26T17:00:00+01:00"
+title = "The Vulkan-o erupts - Citra Vulkan is here!"
+tags = [ "feature-update" ]
+author = "autumnburra"
+coauthor = "sleepingsnake"
++++
+
+Hey there, Citra fans! (We haven't used that one for a while, have we?)
+You've all wanted it and we've heard your cries and pleas. Vulkan has now arrived and is here to stay!
+
+# Why Vulkan?
+
+Since the dawn of time- well, since Citra's creation, many, *many* users have requested we add support for the [Vulkan Graphics API](https://en.wikipedia.org/wiki/Vulkan.); better performance in many cases, better support on devices, especially Android. What is there not to love about it?
+
+{{< sidebyside "image" ""
+    "sm3dlogl.png=FPS: 75 Speed: 125%"
+    "sm3dlvk.png=FPS: 115 Speed: 193%" >}}
+
+But oh boy, was it ever a challenge!
+
+Vulkan support is also becoming a standard feature across many emulators, such as [Dolphin](https://dolphin-emu.org/), [Ryujinx](https://ryujinx.org/), [RPCS3](https://rpcs3.net/), [yuzu](https://yuzu-emu.org/), and many others. 
```

Comment:
I find this paragraph a bit disconnected to the line "But oh boy, was it ever a challenge!".
Maybe we should move this one, or just expand a bit on what the challenges were?